src/java/org/apache/cassandra/service/accord/txn/TxnData.java

```diff
@@ -75,6 +75,7 @@ public Set<Map.Entry<TxnDataName, FilteredPartition>> entrySet()
         return data.entrySet();
     }
 
+    // This is inadequate
```

Comment:
I need to remove this. Accord does single replica reads so no need to merge duplicates.

For live migration Cassandra will probably coordinate quorum reads for Accord so the merging will already be done.